app/uk/gov/hmrc/tariffclassificationfrontend/models/AppealType.scala

```diff
@@ -18,17 +18,20 @@ package uk.gov.hmrc.tariffclassificationfrontend.models
 
 object AppealType extends Enumeration {
   def format(value: AppealType): String = value match {
+    case ADR => "Alternative Dispute Resolution (ADR)"
     case REVIEW => "Review"
     case APPEAL_TIER_1 => "Appeal tier 1"
     case APPEAL_TIER_2 => "Appeal tier 2"
     case COURT_OF_APPEALS => "Court of appeals"
     case SUPREME_COURT => "Supreme Court"
   }
 
+  def heading(value: AppealType): String = value match {
+    case ADR => "Dispute"
+    case REVIEW => "Review"
+    case _ => "Appeal"
+  }
+
   type AppealType = Value
-  val REVIEW  = Value(1)
-  val APPEAL_TIER_1 = Value(2)
-  val APPEAL_TIER_2 = Value(3)
-  val COURT_OF_APPEALS = Value(4)
-  val SUPREME_COURT = Value(5)
+  val REVIEW, APPEAL_TIER_1, APPEAL_TIER_2, COURT_OF_APPEALS, SUPREME_COURT, ADR = Value
```

Comment:
I know it probably doesn't matter but ADR is the first enum value in the backend, but last value here in front end. Should it be consistent?

Reply:
I'll change it.  Sure it doesn't matter though.